update-rules.json

```diff
@@ -829,5 +828,286 @@
       }
     },
     "version_extractor": "^[0-9]{2}"
+  },
+    "winscp": {
+     "url": "https://winscp.net/eng/download.php",
+     "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "#content > h3:nth-child(8)",
+        "baselink": "https://downloads.sourceforge.net/project/winscp/WinSCP/{{.version}}/WinSCP-{{.version}}-Setup.exe"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+    "processhacker": {
+     "url": "http://processhacker.sourceforge.net/downloads.php",
+     "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='setup.exe']"
+      }
+    },
+    "version_extractor": "([0-9].[0-9]{1,4})"
+  },
+   "everything-search": {
+    "url": "https://www.voidtools.com/downloads/",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "a[href$='x86.Multilingual-Setup.exe']"
+      },
+      "x86_64":{
+        "rule_type": "css-link",
+        "selector": "a[href$='x64.Multilingual-Setup.exe']"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "mpc-hc": {
+    "url": "https://mpc-hc.org/downloads/",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "a[href$='.x86.exe']"
+      },
+      "x86_64":{
+        "rule_type": "css-link",
+        "selector": "a[href$='.x64.exe']"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "deluge": {
+    "url": "http://dev.deluge-torrent.org/wiki/Download",
```

Comment:
for Deluge have you tried getting the link directly from the download page (http://download.deluge-torrent.org/windows/?C=M;O=D) where all the links are located? Their file naming convention doesn't seem very constant so I'm a bit nervous about getting the version number only.